### ISSUE TITLE
Headers' Pre-Trained weights

### ISSUE THREAD

**Issue by MiguelEncina** [TaharChettaoui/FRoundation#4]
Hi @TaharChettaoui, impressive work and thanks a lot for sharing!

I'm conducting research for the University of Seville, and we're very interested in your work. I have already tested your FoundPAD pre-trained models, and we would now like to test these.  Would it be possible to upload the pre-trained weights of the classification header as well, as you did with the FoundPAD pre-trained models? Thanks!

**Comment by TaharChettaoui** [comment]
Hello Miguel, thank you for your kind words and interest.

Unlike FoundPAD, FRoundation does not rely on Headers. We fine-tune Foundations models (CLIP/DinoV2) with LoRA to obtain feature representation (output vector) from face images for the face verification task.

**Comment by MiguelEncina** [comment]
My bad! Thanks for the clarification! May I have access to the optimal thresholds obtained during the training of each dataset?

**Comment by TaharChettaoui** [comment]
Could you please clarify? We trained using the CosFace loss, and I do not recall saving any thresholds during training.

**Comment by MiguelEncina** [comment]
Sure! I have assumed that there must be a threshold that determines whether the distance between two images is small enough for them to belong to the same person, or whether it is greater than the threshold and therefore the people in the images are different. During training, I have assumed that this threshold is adjusted to produce better results.

**Comment by TaharChettaoui** [comment]
CosFace uses margin-based softmax loss to enforce that embeddings of the same identity are closer, and embeddings of different identities are pushed further apart. 

During inference, you do define a threshold on the cosine similarity or cosine distance to decide whether two images belong to the same person. The threshold is not adjusted during training but is tuned afterward for best verification performance. These thresholds are tuned during evaluation on the different benchmarks.

**Comment by MiguelEncina** [comment]
Alright, thanks!
